dotnet-sso-example/Views/Home/Callback.cshtml

```diff
@@ -1,7 +1,36 @@
 ﻿@{
-    ViewData["Title"] = "Callback";
+    ViewData["Title"] = "Login Successful";
 }
 
-<h2>Your User Profile</h2>
+<body class="container_success">
```

Comment:
The callback page doesn't appear to render correctly: 
![CleanShot 2022-07-11 at 10 53 46](https://user-images.githubusercontent.com/47123183/178317278-5903176d-4247-44ff-bdef-6e9685c2fae6.png)